packages/framework/src/Model/Product/Parameter/ParameterRepository.php

```diff
@@ -321,7 +320,14 @@ protected function getParameterValuesIndexedByProductIdAndParameterName(
     public function getParametersByUuids(array $uuids): array
     {
         $parametersByUuid = [];
-        $parameters = $this->getParameterRepository()->findBy(['uuid' => $uuids]);
```

Comment:
btw, you can define the ordering as a second parameter of the `findBy` method :slightly_smiling_face: